<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCustomerEdit</class>
 <widget class="QDialog" name="DialogCustomerEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wBg" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>44</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lbTitle">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font:  28px &quot;Microsoft JhengHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>客戶資料設定</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
         <item>
          <widget class="QGroupBox" name="gbox0">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="title">
            <string>基本資料</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QWidget" name="widget_4" native="true">
              <property name="font">
               <font>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>客群:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="cbClass">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;設定好的分類(管理設定-&amp;gt;系統管理-&amp;gt;分類設定)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="lbId">
                 <property name="text">
                  <string>編號:</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>備註:</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="1">
                <widget class="QTextEdit" name="txNote1"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>名稱:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="txName"/>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>等級:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QComboBox" name="cbVip">
                 <item>
                  <property name="text">
                   <string>一般</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>VIP</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>幣別:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QComboBox" name="cbCurrency">
                 <item>
                  <property name="text">
                   <string>新台幣(NTD)</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>港幣(HKD)</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>人民幣(RMB)</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>林吉特(MYR)</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>新加坡元(SGD)</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>後五碼:</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QLineEdit" name="txNum5"/>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="txId"/>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="txLine"/>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Line</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget_2">
           <property name="font">
            <font>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <widget class="LayerCustomerGameEdit" name="page"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QWidget" name="wBottom" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnDel">
           <property name="font">
            <font>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>刪除</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>324</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnCancel">
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnOk">
           <property name="text">
            <string>確定</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LayerCustomerGameEdit</class>
   <extends>QWidget</extends>
   <header>LayerCustomerGameEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbClass</tabstop>
  <tabstop>txNote1</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnDel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
